gensim/models/ldamulticore.py

```diff
@@ -276,13 +275,16 @@ def process_result_queue(force=False):
                 if eval_every > 0 and (force or (self.num_updates / updateafter) % eval_every == 0):
                     self.log_perplexity(chunk, total_docs=lencorpus)
 
+        # HACK np.int32 should be enough for word ids.
+        chunk_dtype = np.dtype([('1', np.int32), ('2', self.dtype)]) if chunks_as_numpy else self.dtype
```

Comment:
What does this construction do? Why the 2-element list, with two different types inside (int32 and self.dtype)?

Too "clever" to be dropped without a comment.